ctk/common/src/test/kotlin/org/codice/compilance/verification/core/SamlVersioningVerifierSpec.kt

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compilance.verification.core
+
+import io.kotlintest.forAll
+import io.kotlintest.matchers.string.shouldContain
+import io.kotlintest.shouldThrow
+import io.kotlintest.specs.StringSpec
+import io.mockk.every
+import io.mockk.mockk
+import org.codice.compliance.Common.Companion.buildDom
+import org.codice.compliance.SAMLComplianceException
+import org.codice.compliance.SAMLCoreRefMessage
+import org.codice.compliance.SAMLCore_4_1_3_2_a
+import org.codice.compliance.SAMLCore_4_1_3_2_b
+import org.codice.compliance.utils.TestCommon.Companion.ASSERTION_NAMESPACE
+import org.codice.compliance.utils.TestCommon.Companion.PROTOCOL_NAMESPACE
+import org.codice.compliance.verification.core.SamlVersioningVerifier
+import java.time.Instant
+
+class SamlVersioningVerifierSpec : StringSpec() {
+    init {
+        val specRefMsg = mockk<SAMLCoreRefMessage>()
+        every { specRefMsg.message } returns "Test message"
+        every { specRefMsg.name } returns "Test"
+
+        val instant = Instant.now()
+        val response = { resVersion: String, resUri: String?, version: String, uri: String? ->
```

Comment:
✏️ I think calling these uri parameters `namespace` would be a little more clear.